drink/src/lib.rs

```diff
@@ -23,17 +23,18 @@ use pallet_contracts::{ContractExecResult, ContractInstantiateResult};
 #[cfg(feature = "session")]
 pub use session::mock::{mock_message, ContractMock, MessageMock, MockedCallResult, Selector};
 /// Export pallets that are used in the minimal runtime.
-pub use {frame_support, frame_system, pallet_balances, pallet_contracts, pallet_timestamp};
+pub use {
+    frame_support, frame_system, pallet_balances, pallet_contracts, pallet_timestamp, paste,
+    sp_externalities::Extension, sp_io::TestExternalities,
+};
 
-pub use crate::runtime::minimal::{self, MinimalRuntime};
-use crate::runtime::AccountIdFor;
+pub use crate::runtime::minimal::{self, MinimalSandbox};
 
 /// Alias for `frame-system`'s `RuntimeCall` type.
-pub type RuntimeCall<Runtime> = <Runtime as frame_system::Config>::RuntimeCall;
+pub type RuntimeCall<R> = <R as frame_system::Config>::RuntimeCall;
 
 /// Alias for `pallet-balances`'s Balance type.
-pub type BalanceOf<Runtime> =
-    <<Runtime as pallet_contracts::Config>::Currency as Inspect<AccountIdFor<Runtime>>>::Balance;
+pub type BalanceOf<R> = <R as pallet_balances::Config>::Balance;
```

Comment:
What w twist (https://github.com/inkdevhub/drink/pull/106/files#r1509471655). @pmikolajczyk41 , are you OK with this change? Since you made the previous one and had some arguments it was "more correct".

Reply:
ah this can go away, I probably missed that in the merge